nexus/Freqlog/Freqlog.py

```diff
@@ -346,8 +357,21 @@ def start_logging(self, new_word_threshold: float | None = None, chord_char_thre
                       f"allowed_chars={self.allowed_chars}, "
                       f"allowed_first_chars={self.allowed_first_chars}, "
                       f"modifier_keys={self.modifier_keys}")
-        self.listener.start()
-        self.mouse_listener.start()
+
+        def log_start():
+            if self.loggable:
+                self.listener = vinput.EventListener(True, True, True)
+
+            try:
+                self.listener.start(
+                    lambda x: self._on_key(x),
+                    lambda x: self._on_mouse_button(x),
+                    lambda x: self._on_mouse_move(x))
+            except vinput.VInputException as e:
+                logging.error("Failed to start listeners: " + str(e))
```

Comment:
Nit: you're only starting one listener.

Reply:
No, there's two: one for the keyboard, and one for mouse buttons.